contracts/Comptroller.sol

```diff
@@ -721,6 +737,14 @@ contract Comptroller is ComptrollerV5Storage, ComptrollerInterface, ComptrollerE
             if (oErr != 0) { // semi-opaque error code, we assume NO_ERROR == 0 is invariant between upgrades
                 return (Error.SNAPSHOT_ERROR, 0, 0);
             }
+
+            // Change Code here *
+            // Check if the collateral is paused, only for borrowing
```

Comment:
Hmm... this comment looks oddly familiar. These next few lines of code too. At least you changed the first line of the comments. 

https://github.com/InverseFinance/anchor/pull/21/commits/29a73d667782483b452f5e17fb2100c3fc4e9068

Reply:
sorry did not meant to steal your thing, this was my first git pull request, just wanted to get a feeling for this. And keep your code on history to check later when I study this stuff

Reply:
Now that I know it's not done with malice it's alright. Shouldn't have assumed the worst in people :P.